modal/image.py

```diff
@@ -429,9 +429,15 @@ def _hydrate_metadata(self, metadata: Optional[Message]):
             assert isinstance(metadata, api_pb2.ImageMetadata)
             self._metadata = metadata
 
-    def _add_mount_layer_or_copy(self, mount: _Mount, copy: bool = False):
+    def _add_mount_layer_or_copy(
+        self,
+        mount: _Mount,
+        *,
+        copy: bool = False,
+        description: str | None = None,
```

Comment:
I think we still need to use `typing.Union` as we're still supporting python 3.9